styles/image.css

```diff
@@ -0,0 +1,30 @@
+.Image__container{
```

Comment:
maybe there is a typo Image make it small "i" it may create problem in future

Reply:
thank you for aknowledge , i will update in next commit